src/Materials/babylon.material.ts

```diff
@@ -466,8 +466,10 @@
         /**
          * Gets a boolean indicating that current material needs to register RTT
          */
-        public hasRenderTargetTextures = false;
-
+        public get hasRenderTargetTextures(): boolean {

```

Comment:
Hello not sure why this is a problem? The child can still override it with a property even if it is not a property at base class level

Reply:
This bug occurs since version 3.3.0-beta.6.
Here is the error : 
![image](https://user-images.githubusercontent.com/5614189/45633000-ff8f9880-ba9e-11e8-8627-3eaadbcc3af3.png)
The parent class define a property that could be overriden by its children. But some of its children have the same property name defined as a getter. So there is a conflict.
In the class child, when we want to set the property, it considers it as a getter and prevent setting a value because the setter doesn't exist.

Reply:
ok makes sense..So good for this change! Thanks!